docs-src/basic_usage.rst

```diff
@@ -311,3 +311,52 @@ Get a list of team members
 See `users.list <https://api.slack.com/methods/users.list>`_ for more info.
 
 .. include:: metadata.rst
+
+--------
+
+Web API Rate Limits
+--------------------
+Slack allows applications to send no more than one message per second. We allow bursts over that
+limit for short periods. However, if your app continues to exceed the limit over a longer period
+of time it will be rate limited.
+
+Here's a very basic example of how one might deal with rate limited requests.
+
+If you go over these limits, Slack will start returning a HTTP 429 Too Many Requests error,
+a JSON object containing the number of calls you have been making, and a Retry-After header
+containing the number of seconds until you can retry.
+
+
+.. code-block:: python
+
+  from slackclient import SlackClient
+  import time
+
+  slack_token = os.environ["SLACK_API_TOKEN"]
+  sc = SlackClient(slack_token)
+
+  # Simple wrapper for sending a Slack message
+  def send_slack_message(channel, message):
+    return sc.api_call(
+      "chat.postMessage",
+      channel=channel,
+      text=message
+    )
+
+  # Make the API call and return save results to `response`
+  response = send_slack_message("#python", "Hello, from Python!")
```

Comment:
do we really want to give an example where channel names are used? we're advising people to use channel IDs as much as possible.

Reply:
I reused it to echo the other channel examples.

Reply:
👌 i guess we'll have to make a larger pass over the docs to stop using that practice.